includes/wc-product-functions.php

```diff
@@ -200,6 +200,40 @@ function wc_get_featured_product_ids() {
 	return $featured_product_ids;
 }
 
+/**
+ * Populate a batch of rating count lookup table data for products.
+ *
+ * @since 4.0.1
+ * @param int $product Product Object.
+ */
+function wc_get_parent_grouped_id( $product ) {
+
+	global $wpdb;
+
+	$group_ids = array();
+
+	if ( $product->is_type( array( 'simple' ) ) ) {
```

Comment:
Grouped products accept any type of product, not only simple products, it also works with variable products, other grouped products, external products and custom type of products included by 3rd party extensions.

So updating only for simple products will not solve all the problem.